main.go

```diff
@@ -78,6 +78,7 @@ func setupFlags(cmd *cobra.Command) {
 	flagSet.StringVar(&cfg.CriConfig.Listen, "listen-cri", "/var/run/pouchcri.sock", "Specify listening address of CRI")
 	flagSet.StringVar(&cfg.CriConfig.NetworkPluginBinDir, "cni-bin-dir", "/opt/cni/bin", "The directory for putting cni plugin binaries.")
 	flagSet.StringVar(&cfg.CriConfig.NetworkPluginConfDir, "cni-conf-dir", "/etc/cni/net.d", "The directory for putting cni plugin configuration files.")
+	flagSet.StringVar(&cfg.CriConfig.SandboxImage, "sandbox-image", "k8s.gcr.io/pause-amd64:3.0", "The image used by sandbox container.")
```

Comment:
Shall we change the default image to a better image which is easier to access in China? 
I think that would be more user-friendly.